bbn-test-5/babylon-node/README.md

```diff
@@ -0,0 +1,283 @@
+
+`test`: Stores keys unencrypted on disk. It’s meant for testing purposes and 
+should never be used in production.
+`file`: Stores encrypted keys on disk, which is a more secure option than test but 
+less secure than using the OS keyring.
+`os`: Uses the operating system's native keyring, providing the highest level of 
+security by relying on OS-managed encryption and access controls.
+
+The `<name>` specifies a unique identifier for the key.
+
+The execution result displays the address of the newly generated key and its 
+public key. Following is a sample output for the command:
+
+```shell
+- address: bbn1kvajzzn6gtfn2x6ujfvd6q54etzxnqg7pkylk9
+name: <name>
+pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey",
+key: "Ayau+8f945c1iQp9tfTVaCT5lzhD8n4MRkZNqpoL6Tpo"}'
+type: local
+```
+
+Make sure to securely store this information, particularly your address and 
+private key details. Losing access to these credentials would mean losing 
+control of your validator and any staked funds.
+
+### Keys for a BLS Voting
+#### What is BLS Voting
+
+BLS (Boneh-Lynn-Shacham) voting is used to create compact checkpoints at the 
+end of each epoch. An epoch is a fixed period within the blockchain, defined or 
+set by a number of blocks, during which a validator set remains consistent. 
+This stability allows Babylon to create and submit checkpoints only once per 
+epoch, reducing the overhead of interacting with the Bitcoin blockchain.
+
+To read more on BLS Voting please see [here](#) # TODO: add link when it is ready 
+
+#### Create BLS Key
+
+The Babylon network uses BLS (Boneh-Lynn-Shacham) signatures as an efficient 
+way to create consensus checkpoints at the conclusion of each epoch. As a validator, 
+you'll need to participate in this process by submitting BLS signatures. 
+This requires a special BLS key pair that's separate from your validator key.
+
+To generate your BLS key, you'll use your validator address from the previous step. 
+Run this command:
+
+```shell
+babylond create-bls-key <address>
+```
+
+Replace `<address>` with your validator address from the earlier keyring 
+generation (it should look similar to `bbn1kvajzzn6gtfn2x6ujfvd6q54etzxnqg7pkylk9`). 
+
+The system will automatically:
+1. Generate a new BLS key pair
+2. Associate it with your validator address
+3. Store it in your node's configuration file at 
+`~/.<path>/config/priv_validator_key.json`
+
+This key will be used automatically by your validator software when it needs 
+to participate in epoch-end signature collection. The BLS signatures help 
+create compact, efficient proofs of consensus that can be verified by other 
+blockchain networks, particularly Bitcoin.
+
+Important: The `priv_validator_key.json` file contains sensitive key material. 
+Make sure to backup this file and store it securely, as it's essential for your 
+validator's operation and cannot be recovered if lost.
+
+## Sync Node
+
+We are now ready to sync the node.
```

Comment:
You do not need to have validator keys to sync the node. The validator paragraph above should move completely to the validators doc.